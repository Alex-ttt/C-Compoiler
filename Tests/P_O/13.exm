for (;;) ..
First action:
    i
  =
    0
Stop condition:
    i
  <
    n
Step:
  i
  ++
Body:
  Statements:
    fo
    (
    )
