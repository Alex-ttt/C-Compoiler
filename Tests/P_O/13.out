 Symbols Table:
   n int
   fo  void function fo(
  )
    {
      << block is empty >>
    }
   main  int function main(
  )
    {
      Statements:
      for (;;) ..
      First action:
          i
        =
          0
      Stop condition:
          i
        <
          n
      Step:
        i
        ++
      Body:
        {
          Statements:
          fo
          (
          )
        }
      Symbols Table:
              i int
    }
